csharp/src/Microsoft.Azure.EventHubs.Processor/PartitionContext.cs

```diff
@@ -20,29 +18,19 @@ internal PartitionContext(EventProcessorHost host, string partitionId, string ev
             this.EventHubPath = eventHubPath;
             this.ConsumerGroupName = consumerGroupName;
             this.ThisLock = new object();
+            this.SequenceNumber = 0;
+            this.Offset = PartitionReceiver.StartOfStream;
```

Comment:
initialized the SequenceNumber and Offset in internal the constructor. Offset constant should be -1. Technically you shouldnt receive null then, right?